client/components/suggestion-search/index.jsx

```diff
@@ -104,6 +111,20 @@ class SuggestionSearch extends Component {
 		this.props.onChange( newValue, true );
 	}
 
+	onSuggestionItemRender = ( { index, suggestionIndex } ) => {
```

Comment:
This is a new prop. It takes an object which is different from `handleSuggestionMouseDown`.

I'd like to hear suggestions on whether this is more broadly useful. Perhaps both should receive an object with `{ suggestion, currentIndex, suggestionIndex }`, but that would require changing the external API of `handleSuggestionMouseDown` 😕

Reply:
Since it's propagated down to the suggestion item's `componentDidMount` -- should we call it
 
```suggestion
	onSuggestionItemMount = ( { index, suggestionIndex } ) => {
```

instead, to be more accurate? :grimacing:

Reply:
(Just to state the obvious, firing an analytics event upon render or mount is kinda weird and smelly; I realize that we can't change it easily here though.)

Reply:
👍 469ac678a328819c73ad4940e3230b1bd18139e5